onnxruntime/test/contrib_ops/attention_op_test.cc

```diff
@@ -29,6 +29,7 @@ static void RunAttentionTest(
     int number_of_heads,
     bool use_float16 = false,
     bool is_unidirectional = false,
+    bool is_input_dimension_swaped = false,
```

Comment:
This test still has swaped instead of swapped. Not sure how the test runs as the "input_dimension_swaped" attribute would now be invalid. Does input_dims need to be updated to account for this too?

Reply:
Sorry I forgot to test locally...
For the input_dims, I think we do not actually have to modify that function(RunAttentionTest), but I changed the variable name in my test for better readability